### PR TITLE
4.4.1 routing

### PR DESCRIPTION
**Pull request by keenangee**
Added routing functionality, also added temp button to toggle on and off. Current car location is hardcoded

### CODE REVIEW

File: Components/TomTomMaps.jsx [hunk 1]
```diff
@@ -47,20 +47,16 @@ export default function TomTomMaps() {
...
   return (
     <View style={styles.container}>
-      <View style={styles.buttons}>
-        <TextInput
-          style={styles.textInput}
-          onChangeText={setMapCenter}
-          value={mapCenter}
-        ></TextInput>
-      </View>
+      {/* Temp button to enable/disable navigation */}
```
Comment (Cloud-Monkey) [inline Components/TomTomMaps.jsx:50]: Should this still be here?

Comment (keenangee) [reply Components/TomTomMaps.jsx:50]: Yeah, left it to show functionality for when the ‘locate car’ button is added

**Review by ah28011992 — APPROVED** [review]
Checked out on local branch and all seems to work well statically! Nice work

File: Components/TomTomMaps.jsx (hunk 1 above)
Comment (Cloud-Monkey) [reply Components/TomTomMaps.jsx:50]: cool, thank you

**Review by Cloud-Monkey — APPROVED** [review]
Looks good

**Pull request merged**